app/models/user.rb

```diff
@@ -15,4 +15,12 @@ class User < ApplicationRecord
   has_secure_password
 
   before_save { email.downcase! }
+
+  class << self
+    def User.digest(string)
```

Comment:
this method inside class self, so dont need User anymore
```suggestion
    def digest string
```